cypress/e2e/awx/resources/inventoryHostRegular.cy.ts

```diff
@@ -0,0 +1,95 @@
+import { Inventory } from '../../../../frontend/awx/interfaces/Inventory';
+import { Organization } from '../../../../frontend/awx/interfaces/Organization';
+import { AwxUser } from '../../../../frontend/awx/interfaces/User';
+import { createAndEditAndDeleteHost, checkHostGroup } from '../../../support/hostsfunctions';
+
+describe('Inventory Host Tab Tests for regular inventory', () => {
+  let organization: Organization;
+  let inventory: Inventory;
+  let user: AwxUser;
+
+  before(() => {
+    cy.awxLogin();
+    cy.createAwxOrganization().then((org) => {
+      organization = org;
+      cy.createAwxInventory({ organization: organization.id }).then((inv) => {
+        inventory = inv;
+      });
+      cy.createAwxUser(organization).then((testUser) => {
+        user = testUser;
+      });
+    });
+  });
+
+  after(() => {
+    cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
+    cy.deleteAwxUser(user, { failOnStatusCode: false });
+    cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
+  });
+
+  it('can create, edit, assosiat and disassosiate groups at inventory -> hosts -> groups tab', () => {
+    // use checkHostGroup function in order to test inventory host group
+    // the test is checking: create, verify, multi assosiate and disassosiate of groups (using bluk)
+    // and single assosiate and disassosiate of group, edit and delete
+    checkHostGroup('inventory_host', organization);
+  });
+
+  it('can create, edit and delete inventory host action from list view', () => {
+    // use createAndEditAndDeleteHost function in order to test inventory hosts basic functions
+    // after navigating to the right url
+    // the test covers create, verify, edit and delete of hosts form inventory
+    cy.visit(`/infrastructure/inventories/inventory/${inventory.id}/details`);
+    cy.clickTab(/^Hosts$/, true);
+    createAndEditAndDeleteHost('inventory_host', inventory);
+  });
+
+  it('can edit and delete inventory host action from details view', () => {
+    //can and delete host from details view
```

Comment:
Missing implementation? If not, please skip this test.

Reply:
done